src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -109,11 +127,32 @@ public void InitializeService(ServiceHost serviceHost)
                     null,
                     null);
                 }
+            }
+
+            return dataContainer;
+        }
+
+        /// <summary>
+        /// Handles a backup request
+        /// </summary>
+        internal static async Task HandleBackupRequest(
```

Comment:
>HandleBackupRequest [](start = 35, length = 19)

could you please add some unit tests for the new messages added as part of this review?